### PR TITLE
feat(labelSelector): option matching Ingresses based on labelSelectors

### LINKED ISSUE #264: labelSelector field for kuberneteingress provider
labelSelector allows to decide wheter an ingress should be managed by traefik or not (ingress controller selection). Right now, the only to achieve this is through the additionalArguments, but should rather be available directly under the provider section, such as:

```
providers:
  kubernetesIngress:
    enabled: true
    labelSelector: A and not B
```

### PR DESCRIPTION
**Pull request by faust64**
Fixes #264

traefik can pick Ingress objects based on a labelSelector.
This can be useful in larger clusters, when ingress are dedicated to an environment (prod, stage, ...), when SDN is broken into security zones with dedicated ingresses, ...

### CODE REVIEW

**Comment by SantoDE** [comment]
Hey @faust64,

can we have some quick unit tests for that please? :)

File: traefik/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: traefik
 description: A Traefik based Kubernetes ingress controller
 type: application
-version: 9.15.2
+version: 9.15.5
```
Comment (SantoDE) [inline traefik/Chart.yaml:5]: 9.16.0 plx :)

**Review by SantoDE — CHANGES_REQUESTED** [review]
One last step

File: traefik/Chart.yaml (hunk 1 above)
Comment (faust64) [reply traefik/Chart.yaml]: :+1:

**Comment by faust64** [comment]
See #390

**Pull request closed without merging**